addon/src/lib/environments.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
   any: { name: 'any', baseUrl: '*', whitelist: '*' },
   local: {
     name: 'local',
-    baseUrl: 'http://testpilot.dev:8000',
+    baseUrl: 'https://example.com:8000',
```

Comment:
Should we get rid of the example environment then, if local is going to use example?